Configuration/StandardSequences/python/Generator_cff.py

```diff
@@ -55,7 +55,10 @@
 GeneInfo = cms.Sequence(GeneInfoTask)
 genJetMET = cms.Sequence(genJetMETTask)
 
-pgen = cms.Sequence(cms.SequencePlaceholder("randomEngineStateProducer")+VertexSmearing+GenSmeared+GeneInfo+genJetMET)
+from SimPPS.Configuration.GenPPS_cff import *
+PPSTransport = cms.Sequence(PPSTransportTask)
+
+pgen = cms.Sequence(cms.SequencePlaceholder("randomEngineStateProducer")+VertexSmearing+GenSmeared+GeneInfo+genJetMET+PPSTransport)
```

Comment:
This could be just
```suggestion
pgen = cms.Sequence(cms.SequencePlaceholder("randomEngineStateProducer")+VertexSmearing+GenSmeared+GeneInfo+genJetMET, PPSTransportTask)
```

Reply:
Can I use the "commit suggestion" option shown in the message?

Reply:
> Can I use the "commit suggestion" option shown in the message?

Yes, you can use it if you want.